connect/connect-c8/element-template-c8/src/main/java/io/miragon/miranum/connect/adapter/in/c8/elementtemplates/CamundaC8ElementTemplateConverter.java

```diff
@@ -0,0 +1,31 @@
+package io.miragon.miranum.connect.adapter.in.c8.elementtemplates;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import io.miragon.miranum.connect.adapter.in.c8.elementtemplates.schema.CamundaC8ElementTemplate;
+
+public class CamundaC8ElementTemplateConverter
+{
+
+    private static final String $SCHEMA = "https://unpkg.com/@camunda/zeebe-element-templates-json-schema@0.8.0/resources/schema.json";
```

Comment:
I would not use special characters in variable definitions. May it make sense to provide this URL as property in application properties?

Reply:
❓ I'm not shure how to do that. For my understanding this class is not managed by Spring so how can I inject a property?

Reply:
You can leave this out for now. We can have a look after refactoring it to be part of the maven plugin

Reply:
You can leave this out for now. We can have a look at it after refactoring it to be part of the maven plugin